src/startup_scripts.rs

```diff
@@ -591,8 +591,6 @@ run_delegate_stake() {
     fi
     echo "delegated stake"
   fi
-
-  solana --url $LOAD_BALANCER_RPC_URL --keypair $IDENTITY_FILE stakes validator-accounts/stake.json
```

Comment:
yeah. the stakes command is weird here. we can remove it or use stake-account here. (https://github.com/anza-xyz/agave/pull/1284)